### DISCUSSION TITLE
Why pencil tool pointer and the actual lines drawn has a considerable distance?

### DISCUSSION THREAD

**Discussion by jijojosephk** [phase1geo/Annotator#96] [Q&A]
I hope this image is enough to understand the problem. Drawing is not happening where it is supposed to be.

![image](https://github.com/phase1geo/Annotator/assets/11662148/d6ad4723-2c6a-4073-bbbd-36deb2f05644)

**Comment by phase1geo** [comment]
I have converted this question to an issue.  See #97 for further communication on this issue.  I have a couple of outstanding questions and a request posted on that issue currently.
